runtime/oti/j9nonbuilder.h

```diff
@@ -2030,64 +2030,21 @@ typedef struct J9BCTranslationData {
 	UDATA structureUnusedButConstantsUsedInDDR;
 } J9BCTranslationData;
 
-#define BCT_PcFlagBranchFlipped  0x4000000
-#define BCT_MajorClassFileVersionMask  0xFF000000
-#define BCT_ActionBreakpoint  7
-#define BCT_InitCopyLongs  17
+#define BCT_LittleEndianOutput  0
 #define BCT_BigEndianOutput  1
-#define BCT_ActionReturn  4
-#define BCT_PcFlagLookupSwitch  0x800000
-#define BCT_ErrLocalsMapTooSmall  10
-#define BCT_MajorClassFileVersionMaskShift  24
-#define BCT_ErrIllegalState  6
-#define BCT_InitCopyIntegersW  14
-#define BCT_ErrConstantPoolMapInvalid  4
```

Comment:
Are these really all unused?  I thought some?  all? of the BCT_Err flags were related to verbose verification

Reply:
I searched the codebase for each constant that was removed. I needed a new bit in the flags, and it was less than clear what bits were in use, hence the cleanup.

Reply:
Fair enough.  And the testing passes so I can't argue :)